File: src/main/java/com/alxnns1/mobhunter/entities/fish/renderer/SushifishRenderer.java
package com.alxnns1.mobhunter.entities.fish.renderer;

import com.alxnns1.mobhunter.MobHunter;
import com.alxnns1.mobhunter.entities.fish.SushifishEntity;
import com.alxnns1.mobhunter.entities.fish.model.SushifishModel;
import net.minecraft.client.renderer.entity.EntityRendererManager;
import net.minecraft.client.renderer.entity.MobRenderer;
import net.minecraft.util.ResourceLocation;
import net.minecraftforge.api.distmarker.Dist;
import net.minecraftforge.api.distmarker.OnlyIn;

import javax.annotation.Nonnull;

@OnlyIn(Dist.CLIENT)
public class SushifishRenderer extends MobRenderer<SushifishEntity, SushifishModel<SushifishEntity>> {
	public SushifishRenderer(EntityRendererManager renderManagerIn) {
		super(renderManagerIn, new SushifishModel<>(), 0.5f);
	}

	@Nonnull
	@Override
	public ResourceLocation getEntityTexture(@Nonnull SushifishEntity entity) {
		return new ResourceLocation(MobHunter.MOD_ID, "textures/entity/sushifish.png");
	}
}

File: src/main/kotlin/com/alxnns1/mobhunter/entity/fish/renderer/SushifishRenderer.kt
package com.alxnns1.mobhunter.entity.fish.renderer

import com.alxnns1.mobhunter.MobHunter
import com.alxnns1.mobhunter.entity.MHRenderer
import com.alxnns1.mobhunter.entity.fish.SushifishEntity
import com.alxnns1.mobhunter.entity.fish.model.SushifishModel
import net.minecraft.client.renderer.entity.EntityRendererManager
import net.minecraft.util.ResourceLocation

private val RL = ResourceLocation(MobHunter.MOD_ID, "textures/entity/sushifish.png")

class SushifishRenderer(rendererManager: EntityRendererManager) : MHRenderer<SushifishEntity, SushifishModel>(rendererManager, SushifishModel(), 0.5F) {

	override fun getEntityTexture(entity: SushifishEntity): ResourceLocation = RL
}
